macosx/NSImageAdditions.h

```diff
@@ -9,8 +9,4 @@
 + (NSImage*)discIconWithColor:(NSColor*)color insetFactor:(CGFloat)insetFactor;
 - (NSImage*)imageWithColor:(NSColor*)color;
 
-/* macOS < 11 compatibility */
-+ (NSImage*)systemSymbol:(NSString*)symbolName withFallback:(NSString*)fallbackName;
```

Comment:
I suggest removing just `withFallback:` part.
So we don't need to supply `nil` to `accessibilityDescription:` every time, just like before.

Reply:
I'd prefer an incentive to start providing accessibility descriptions where relevant instead. But you decide: do you strongly prefer to keep that wrapper?

Reply:
I personally have no strong opinion on this one, but lean towards using the native call to encourage accessibility descriptions

Reply:
Marking as resolved since @livings124 has approved the PR but the PR can't be merged until all conversations are resolved :slightly_smiling_face: